src/commands/service/pipeline.ts

```diff
@@ -228,9 +239,14 @@ export const execute = async (
 export const commandDecorator = (command: commander.Command): void => {
   buildCmd(command, decorator).action(
     async (serviceName: string, opts: CommandOptions) => {
-      await execute(serviceName, opts, async (status: number) => {
-        await exitCmd(logger, process.exit, status);
-      });
+      await execute(
+        serviceName,
+        process.cwd(),
```

Comment:
It's seems strange to add a `projectPath` param at the `execute` command level if it will always be essentially hardcoded. We appear to be using `projectPath` to find the `bedrock.yaml` file. Is there a better way? cc @mtarng